recipes/r-phylolm/meta.yaml

```
@@ -0,0 +1,82 @@
+{% set version = '2.6.2' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-phylolm
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/phylolm_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/phylolm/phylolm_{{ version }}.tar.gz
+  sha256: d6fa45f58689cce9fe91519ab55a61c1e549e5a7f18b6989bf86234d483cff12
+
+build:
+  merge_build_host: True  # [win]
+  skip: True  # [r_base != "3.6.3"]
```

This skips the R4.0 build. Change to:

```suggestion
  skip: True  # [r_base != "3.6.3"]
```